Vagrantfile

```diff
@@ -27,6 +27,6 @@ Vagrant.configure(2) do |config|
   end
 
   config.vm.provision "shell", inline: <<-SHELL
-    sudo yum install -y git livecd-tools
+    sudo yum install -y git livecd-tools docker parted
```

Comment:
Adding a note on requirement of 'docker' would help to user to understand its usage.